ec2/spark_ec2.py

```diff
@@ -1082,11 +1109,11 @@ def real_main():
                     time.sleep(30)  # Yes, it does have to be this long :-(
                     for group in groups:
                         try:
-                            conn.delete_security_group(group.name)
-                            print "Deleted security group " + group.name
+                            conn.delete_security_group(group_id=group.id)
+                            print "Deleted security group %s, id %s" % (group.name, group.id)
                         except boto.exception.EC2ResponseError:
                             success = False
-                            print "Failed to delete security group " + group.name
+                            print "Failed to delete security group %s, id %s" % (group.name, group.id)
```

Comment:
Same here. No need to print the group ID if it's not something the user interacts with.